src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -340,7 +340,7 @@ const PersonalDetails = (() => {
                 }
                 const get_settings        = data.get_settings;
                 const is_tax_req          = +State.getResponse('landing_company.config.tax_details_required') === 1;
-                const is_for_mt_financial = /real_vanuatu_(standard|advanced)/.test(redirect_url);
+                const is_for_mt_financial = /real_svg_standard/.test(redirect_url);
```

Comment:
and I think this should change to `real_vanuatu_advanced`, we still have that group for financial, right?